distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Day_Of_Week.enso

```diff
@@ -4,22 +4,31 @@ from project.Data.Ordering import all
 
 polyglot java import java.time.DayOfWeek
 
+## Represents a day of the week.
 type Day_Of_Week
+    ## Create a Day_Of_Week indicating Sunday.
```

Comment:
```suggestion
    ## Day_Of_Week representing Sunday.
```

I'd rephrase these.

It's not necessarily `Create` because the constructor may also be used in a pattern-match where it is **de**constructing a value, not creating it.

Very subjective: `indicating` sounds to me like about an error (it's the phrasing we use in many error doc comments), `representing` sounds a bit less loaded IMO.